save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ProjectController.kt

```diff
@@ -147,22 +148,24 @@ class ProjectController(
             projectPermissionEvaluator.hasPermission(authentication, it, Permission.READ)
         }
 
-    @GetMapping("/get/not-deleted-projects-by-organization")
+    @GetMapping("/get/projects-by-organization-and-status")
     @RequiresAuthorizationSourceHeader
     @PreAuthorize("permitAll()")
     @Operation(
         method = "GET",
-        summary = "Get non-deleted projects by organization name.",
-        description = "Get non-deleted projects by organization name.",
+        summary = "Get projects by organization name.",
+        description = "Get projects by organization name.",
     )
     @Parameters(
         Parameter(name = "organizationName", `in` = ParameterIn.PATH, description = "name of an organization", required = true),
+        Parameter(name = "status", `in` = ParameterIn.QUERY, description = "organization filters", required = true),
```

Comment:
Parameter `status` of type `ProjectStatus` described as "organization filters".

Are you sure?

Reply:
Changed